modules/react/pagination/stories/pagination.stories.mdx

```diff
@@ -91,50 +35,45 @@ yarn add @workday/canvas-kit-react
 
 ## Usage
 
-`Pagination` is a compound component, meaning it can be put together and used in a variety of
-different ways. We'll first cover [basic usage](#basic-usage) examples followed by
-[advanced usage](#advanced-usage). Each of the examples is intended to be standalone, so you can
-skip to the section that best fits your use case.
-
-These examples are designed to provide copy-and-paste snippets to help you get up and running
-quickly without much explanation. Further detail on individual component props, behavior, and
-accessibility can be found in the [Components](#components) section.
+### Basic Example
 
----
+`Pagination` includes a container `Pagination` component and a number of subcomponents which can be
+composed in a variety of ways.
 
-### Basic Usage
+In this example, we set up a basic `Pagination` component with the default range of 5 pages, as well
+as step controls (`Pagination.StepToPreviousButton` and `Pagination.StepToNextButton`) that allow
+you to move to the next page or the previous page.
 
-Below are examples we expect will work for most use cases. Please refer to the
-[Advanced Usage](#advanced-usage) section for details on supporting custom implementations.
+<ExampleCodeBlock code={Basic} />
 
-#### Step Controls
+### Hoisted Model
 
-In this example, the component uses step controls (`Pagination.StepToPreviousButton` and
-`Pagination.StepToNextButton`) that allow you to move to the next page and previous pages.
+By default, `Pagination` will create and use its own [model](#model) internally. Alternatively, you
+may configure your own model with `usePaginationModel` and pass it to `Pagination` via the `model`
+prop. This pattern is referred to as
+[hoisting the model](/getting-started/for-developers/resources/compound-components/#configuring-a-model)
```

Comment:
It's a little odd that the title for this section is `Configuring a Model` when we call out "hoisting the model" here. I'm not settled on a solution though.